frontend/jest-puppeteer.config.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     // https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/README.md#jest-puppeteerconfigjs
     browserContext: "incognito",
     server        : {
-        command : "yarn start-server",
+        //command : "yarn start-server",    // SignalR won't work here, as it's just static serving
+        command : "yarn dotnet-serve",
```

Comment:
This doesn't work with the current CI setup, as the .NET assemblies aren't built.

Modify the SignalR registration so that it first checks the server connection, and if available connect to the hub. 

That way local e2e testing is possible, and CI will be happy too. 
Furthermore run the SignalR tests with the server tests.